xcode/Subconscious/Shared/Components/Common/Profile/UserProfileView.swift

```diff
@@ -80,9 +80,9 @@ struct FollowTabView: View {
         ForEach(store.state.following) { follow in
             StoryUserView(
                 story: follow,
-                action: { _ in
+                action: { address in
                     notify(
-                        .requestNavigateToProfile(follow.user.address)
+                        .requestNavigateToProfile(address)
```

Comment:
Extra bug I caught while investigating. This could result in using a non-optimal address to reach a peer before.